packages/slider/src/mwc-slider-base.ts

```diff
@@ -137,14 +146,18 @@ export class SliderBase extends FormElement {
       computeBoundingRect: () => this.mdcRoot.getBoundingClientRect(),
       getTabIndex: () => this.mdcRoot.tabIndex,
       registerInteractionHandler:
-          <K extends EventType>(type: K, handler: SpecificEventListener<K>) =>
-              this.mdcRoot.addEventListener(type, handler),
+          <K extends EventType>(type: K, handler: SpecificEventListener<K>) => {
+            const init = type === 'touchstart' ? {passive: true} : {};
```

Comment:
This should be using `applyPassive()` from `@material/dom/events` like we use in ripple-directive: https://github.com/material-components/material-components-web-components/blob/master/packages/ripple/src/ripple-directive.ts#L17